common/common.cc

```diff
@@ -59,6 +59,20 @@ void sorbet::FileOps::write(string_view filename, const vector<sorbet::u1> &data
     throw sorbet::FileNotFoundException();
 }
 
+void sorbet::FileOps::createDir(string_view path) {
+    auto err = mkdir(string(path).c_str(), S_IRWXU | S_IRWXG | S_IROTH | S_IXOTH);
+    if (err && errno != EEXIST) {
```

Comment:
Is this too permissive? Happy to change the API here.

Reply:
+1, I'd prefer to have two separate functions, one that checks that Dir exists and another that creates and errors if it was already there.

Reply:
created `dirExists`